### DISCUSSION TITLE
Fill color in Area Series

### DISCUSSION THREAD

**Discussion by piyushsinha24** [airbnb/visx#1118] [Q&A]
I used this: 

`<AnimatedAreaSeries dataKey="Supply" data={Supply} {...accessors} fill={"rgb(248, 228, 53)"}/>`

The rgb value shows in the body but I get a diff colour stroke on top by default. How to get rid of that default bluish color? Its kind of urgent. Please help asap.

![Screenshot 2021-03-23 at 9 48 42 PM](https://user-images.githubusercontent.com/26529503/112181026-0e340900-8c22-11eb-94fb-8f6db7ac4a99.png)

**Comment by piyushsinha24** [comment]
Figured out the solution. stroke color can be set using line props. Still I have one more issue. How to match glyph color with Area Series color. Please guide asap.

**Reply by williaster** [reply]
Hey @piyushsinha24, another approach to setting colors is by defining the colors in the chart theme. This can then be used by all of the components including tooltip glyphs. You can see an example [here](https://codesandbox.io/s/themeprovider-sbdvz?file=/Example.tsx), where the colors array will be mapped to series `dataKey`s on the bars and in the tooltip.

If you don't want to define a full on theme, you could extend the light theme and just override the colors like

```tsx
import { lightTheme, XYChart } from '@visx/xychart'`;

const customTheme = {
  ...lightTheme,
  // order should match XYChart child series render order 
  colors: [...myColors],
};

// render
() => (
  <XYChart theme={customTheme}>{...}</XYChart>
)
```

**Reply by piyushsinha24** [reply]
Thanks for the reply. It works just fine.

**Reply by VigneshTrantor** [reply]
Thank you so much @williaster . This also helped to solve the tooltip color issue I was facing for the last 3 days.

**Reply by d3x42** [reply]
Thank you for the solution (it helps me with tooltip dots). Would be great to see it as part of the doc!
